slices/libgdiplus.yaml

```diff
@@ -0,0 +1,27 @@
+package: libgdiplus
+
+essential:
+  - libgdiplus_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libcairo2_libs
+      - libexif12_libs
+      - libfontconfig1_libs
+      - libgif7_libs
+      - libglib2.0-0t64_libs
+      - libharfbuzz0b_libs
+      - libjpeg-turbo8_libs
```

Comment:
```suggestion
      - libjpeg8_libs
```

Reply:
AFAIK this is only for the same layering as the package itself :)

We can change, physically it will be the same filesystem + 1 copyright file, right?

Reply:
Yup. You are right. But it's also to make things more future-proof. If some day `libjpeg8` includes other stuff in addition to `libjpeg-turbo8`, adding `lijpeg8_libs` here will avoid breaking things.